admin/acceptance.py

```diff
@@ -91,6 +91,7 @@ def check_result(f):
             FLOCKER_ACCEPTANCE_CONTROL_NODE=control_node.address,
             FLOCKER_ACCEPTANCE_AGENT_NODES=':'.join(
                 node.address for node in agent_nodes),
+            FLOCKER_ACCEPTANCE_VOLUME_BACKEND='loopback',
```

Comment:
What's the impact of this on our ability to run acceptance tests against the ZFS backend on BuildBot (in the Vagrant configuration, at least)?